deploy/terraform/network.tf

```diff
@@ -75,46 +75,46 @@ resource "aws_security_group" "nia_gp2gp_dmz" {
 
   ingress {
     from_port   = var.http_server_port
-    protocol    = "tcp"
+    protocol    = var.vpc_protocol_default
     to_port     = var.http_server_port
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks = [var.allow_all_ipv6_cidr_block]
```

Comment:
I think it should be var.allow_all_cidr_block instead of var.allow_all_ipv6_cidr_block as originally it was "0.0.0.0/0"

Reply:
oh yes, good catch! Corrected